src/providers/Project/projectHooks.ts

```diff
@@ -9,6 +9,7 @@ import { createDefaultProject, createLocalProject } from './projectDefault';
 import { PROJECT_SERIALIZATION_KEY } from './projectMutator';
 
 function formatProject(project: Project) {
+  if (!project) return project;
```

Comment:
should we return some static value, like `null`?

Reply:
wasn't sure if `null` or `undefined` so return what has been passed in. Also the method already returns project that has been passed in.